apps/backoffice-v2/src/pages/Entity/components/Case/actions-variants/DefaultActions/DefaultActions.tsx

```diff
@@ -29,14 +28,14 @@ export const DefaultActions = () => {
   } = useDefaultActionsLogic();
 
   return (
-    <div className={`flex items-center space-x-4 self-start pe-[3.35rem]`}>
+    <div className={`flex flex-wrap items-center gap-4 self-start pe-[3.35rem]`}>
```

Comment:
Is this for the text on the buttons?

Reply:
Nope, its just gap for containers, as previous fix for small screens.